apps/src/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon.tsx

```diff
@@ -13,6 +13,12 @@ export interface FontAwesomeV6IconProps extends AriaAttributes {
    *  * thin - 100
    * */
   iconStyle?: 'solid' | 'regular' | 'light' | 'thin';
+  /** Icon family
+   *    * brands - for FA brand icons
+   *    * duotone - for FA duotone icons
+   *    * kit - for our custom FA kit icons
+   * */
+  iconFamily?: 'brands' | 'duotone' | 'kit';
```

Comment:
Is "kit" a standard name for custom icons? Why not "custom"?

Reply:
yep, it's set up by font awesome I believe since it actually is an icons from our FA kit.
I've took this from Font Awesome website from our custom icons page, almost same way as we got naming for other icons

Reply:
oh got it sorry I misunderstood how this is used, sounds good!